app.py

```diff
@@ -40,11 +40,13 @@ def handle_chat(incoming):
 
     `incoming` is `{'body': 'the message content'}`.
     """
-    outgoing = {
-        'id': request.sid,
-        'body': sanitize(incoming['body']),
-    }
-    socketio.emit('chat', outgoing)
+    trimmed_message = incoming['body'].strip()
+    if len(trimmed_message) > 0:
```

Comment:
This could be simplified to
```if trimmed_message:```
empty strings in python are "falsy"; so there's no need to check the length

Reply:
Thanks, fixed.